src/libostree/ostree-repo.c

```diff
@@ -2791,18 +2791,14 @@ ostree_repo_load_file (OstreeRepo         *self,
         }
       if (out_file_info)
         {
-          *out_file_info = _ostree_header_gfile_info_new (stbuf.st_mode, stbuf.st_uid, stbuf.st_gid);
-          if (S_ISREG (stbuf.st_mode))
-            {
-              g_file_info_set_size (*out_file_info, stbuf.st_size);
-            }
-          else if (S_ISLNK (stbuf.st_mode))
+          *out_file_info = _ostree_stbuf_to_gfileinfo (&stbuf);
+          if (S_ISLNK (stbuf.st_mode))
             {
               g_file_info_set_size (*out_file_info, 0);
```

Comment:
We don't need this anymore, right? (Or if it's not 0 by default, then should we make this part of `_ostree_stbuf_to_gfileinfo()` ?)